index.html

```diff
@@ -0,0 +1,530 @@
+                </div>
+                <div class="header__profile">
+                    <a href="">
+                        <img src="./img/Profile.svg" alt="LOGO">
+                    </a>
+                </div>
+            </div>
+        </div>
+
+    </header>
+    <div></div>
+    <div class="paint-background"></div>
+    <div class="content">
+
+        <div class="content__catalog">
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/Smartphone.png" alt="PHONES">
+                <p>Телефоны</p>
+
+            </a>
+
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/Computer.png" alt="PHONES">
+                <span>Компьютеры</span>
+            </a>
+
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/Headphones.png" alt="PHONES">
+                <span>Наушники</span>
+            </a>
+
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/TV.png" alt="PHONES">
+                <span>Телевизоры</span>
+            </a>
+
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/Watch.png" alt="PHONES">
+                <p class="card__text">
+                    <span>Часы</span>
+
+                </p>
+            </a>
+
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/Tablet.png" alt="PHONES">
+                <span>Планшеты</span>
+            </a>
+
+            <a class="item-top-cards text-normal-default" href="">
+                <img src="./img/Accessories.png" alt="PHONES">
+                <span>Аксессуары</span>
+            </a>
+        </div>
+
+        <div class="content__sales-title text-normal-huge">
+            Осталось скидки
+            <div class="content__popular-title__timer text-normal-large">
+                06:35
+            </div>
+        </div>
+
+        <div class="content__sales">
+
+            <div class="">
+                <button type="button" class="arrow-button arrow-button-left text-normal-default">
+                    <img src="./img/left_arrow.svg">
+                </button>
+            </div>
+            <div class="item-card">
+                <a href="" class="item-card__top">
+
+                    <div class="text-normal-small content__sales-image">
+                        <img class="" src="./img/items_photo/iPhone.png" alt="photo">
+                        <div class="content__sales-value">
+                            12%
+                        </div>
+                    </div>
+                    <div class="item-card__textbox">
+                    <span class="text-normal-large">
+                        167990₽
+                    </span>
+                        <span class="text-normal-default item-card__text-strikethrough">
+                        167990₽
```

Comment:
В макетах есть отступ перед знаком рубля, можно попробовать хотя бы так 
```suggestion
                        167990 ₽
```

Reply:
убрал в макете отступ